### PR TITLE
Updated vendoring for inclusion of http auth

### PR DESCRIPTION
**Pull request by Conorbro**

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -55,10 +55,10 @@
...
-			"checksumSHA1": "lxe3NRr1SukdE8Mf+xNGcTUR0NI=",
+			"checksumSHA1": "4TLgSCgJZuS5gtytxNvcVk4h8/g=",
 			"path": "github.com/prometheus/common/config",
-			"revision": "167b27da48d058a9b46d84b834d67f68f0243f67",
-			"revisionTime": "2016-03-18T12:23:18Z"
+			"revision": "0866df4b85a18d652b6965be022d007cdf076822",
+			"revisionTime": "2017-06-21T12:10:01Z"
 		},
 		{
 			"checksumSHA1": "Zsc9IQzQDkOzqiAITqxEm0JXdws=",
```
Comment (SuperQ) [inline vendor/vendor.json]: Please update all `github.com/prometheus/common/...` vendoring so everything within that repo matches.

**Comment by SuperQ** [comment]
Odd, it's complaining about the vendoring of `github.com/Sirupsen/logrus`.  Possibly due to the caps change.

File: http.go [hunk 2]
```diff
@@ -123,7 +124,7 @@ func probeHTTP(target string, module Module, registry *prometheus.Registry) (suc
 		Timeout: module.Timeout,
 	}
 
-	tlsconfig, err := module.HTTP.TLSConfig.GenerateConfig()
+	tlsconfig, err := config.NewTLSConfig(&module.HTTP.TLSConfig)
```
Comment (brian-brazil) [inline http.go]: This should be taken care of when you use the NewHTTPClientFromConfig

**Comment by grobie** [comment]
You should remove Sirupsen from the vendor folder and vendor the lowercase Sirupsen now.

**Comment by Conorbro** [comment]
Should be good to go now.

**Comment by SuperQ** [comment]
:+1: Thanks!

**Pull request merged**